eng/config/PublishData.json

```diff
@@ -202,8 +202,8 @@
         "NonShipping"
       ],
       "vsBranch": "main",
-      "insertionCreateDraftPR": false,
-      "insertionTitlePrefix": "[dNext]"
+      "insertionCreateDraftPR": true,
```

Comment:
Again, am not sure we should be creating draft PRs for VS main since it is now equivalent to the previous main-vs-deps (with community being the new equivalent for the old main). cc @RikkiGibson

Reply:
Created https://github.com/dotnet/razor/pull/12127 for the same change in dotnet/razor.